testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/expression/Utils.java

```diff
@@ -44,20 +44,23 @@
     public static void setProperty(String name, String value, ModelControllerClient client) {
         ModelNode modelNode = createOpNode("system-property=" + name, ADD);
         modelNode.get(VALUE).set(value);
+        modelNode.get(OPERATION_HEADERS).get(ALLOW_RESOURCE_SERVICE_RESTART).set(true);
         ModelNode result = executeOp(modelNode, client);
         log.debugf("Added property %s, result: %s", name, result);
     }
     
     public static void removeProperty(String name, ModelControllerClient client) {
         ModelNode modelNode = createOpNode("system-property=" + name, REMOVE);
         ModelNode result = executeOp(modelNode, client);
+        modelNode.get(OPERATION_HEADERS).get(ALLOW_RESOURCE_SERVICE_RESTART).set(true);
```

Comment:
This looks like cruft. SystemPropertyRemoveHandler does not respect this header, and in any case removing the props used in the test shouldn't trigger reload-required. If they do there's a bug.